packages/config/src/index.ts

```diff
@@ -43,7 +43,7 @@ class TruffleConfig {
 
   private eventManagerOptions(
     options: TruffleConfig | Partial<Pick<TruffleConfig, "quiet" | "logger" | "subscribers">>
-  ): any {
+  ): Partial<TruffleConfig> {
```

Comment:
Sorry, I know I already hit approve, but something seems to have gotten mixed up here, and apparently I missed it.

When I was talking about using `Pick`, I was talking about the *return* type (that being the type that needed to change, since previously it was `any`).  Using `Partial<TruffleConfig>` seems pretty underspecified.  I don't know why you would use `Pick` for the input type; that seems needlessly overcomplicated.  Using `TruffleConfig` seems fine for the input type; I had no objection to that.  It's the return type that was a problem, because it was `any`, and that's where I think the `Pick` belongs.

Reply:
Yeah, I wrestled with the TS compiler a while but couldn't get it to like the return type. Maybe it will like the input type being `Partial<TruffleConfig>` and the return type to be the one specified for the arguments.

Reply:
Updated!